integration-test/tests/cms/footer.test.ts

```diff
@@ -0,0 +1,34 @@
+import {expect, request, test} from '@playwright/test';
+import ApiHost from "../ApiHost";
+
+test('GET /api/cms/v1/footer returns correct footer data', async () => {
+  const apiHost = ApiHost.get();
```

Comment:
I think that we can just use built-in fixture `request` that respects configuration options like `baseURL` we specified in playwright config and it is ready to send some requests.
In this case - one line of code should be enough:
```
const response = await request.get('api/cms/v1/footer');
```
Then we can make assertions:
```
expect(response.status()).toBe(200);
expect(body).toEqual(expectedResponse);
```

Reply:
ual, that is better. 

Do you know how can I setup this `baseURL`?

Reply:
About features, could check just the type instead of the real value? because basically we only need to check for the mandatory fields and the type, but not really the value, because it can change.